RubeusCore/Include/level.h

```diff
@@ -55,7 +55,7 @@ namespace Rubeus
 		 *
 		 * @brief	Destructor
 		 */
-		~RLevel();
+		virtual ~RLevel();
```

Comment:
This was the main problem. Virtual destructors are necessary for dependency clearing